apps/webcomponents/src/app/components/gn-record-view/gn-record-view.component.html

```diff
@@ -4,65 +4,75 @@
     [showContent]="!!record?.title"
   >
     <div class="font-title text-[28px] line-clamp-3 mb-4">
-      {{ record.title }}
+      {{ record?.title || 'No Title Available' }}
```

Comment:
The title is a mandatory field and we can assume that it's truthy since otherwise the content ghost should still show